src/olympia/files/models.py

```diff
@@ -377,20 +377,45 @@ def get_localepicker(self):
         statsd.timing('files.extract.localepicker', (end * 1000))
         return res
 
-    @amo.cached_property
+    @property
     def webext_permissions(self):
+        """Return permissions with descriptions in defined order:
```

Comment:
Couldn't we sort the permissions in that order at upload time, and not re-sort them everytime we want to display them ?